legal-api/src/legal_api/services/document_meta.py

```diff
@@ -230,6 +232,28 @@ def get_corrected_ia_reports(self, filing: dict):
 
         return reports
 
+    def get_transition_reports(self):  # pylint: disable=no-self-use
+        """Return transition meta object(s)."""
+        reports = []
+
+        reports.append(
```

Comment:
I think you will have to add some label like ' - Pending' to the Transition Application if it is paid because the filing is pending processing. Please check with Linda.